web/components/GigInfo/GigApplications.js

```diff
@@ -45,24 +49,25 @@ function GigApplications({ applications }) {
                           {
                             'bg-yellow-500': item.status === 'pending',
                             'bg-green-500': item.status === 'approved',
+                            'bg-red-700': item.status === 'dismissed',
                           }
                         )}
                       ></span>
                       <div className="">Applcation {item.status}</div>
                     </div>
                     <div className="lg:w-1/3 lg:text-right w-full text-left">
                       {formatDistance(
-                        subDays(new Date(item.date), 3),
+                        subDays(new Date(item.createdAt), 3),
                         new Date()
-                      )}
+                      )}{' '}
```

Comment:
instead of using a space wrap it with a span with margin

Reply:
fixed